scripts/requirements-build-test.txt

```diff
@@ -21,3 +21,6 @@ pytest
 
 # used for mocking functions in pytest
 mock
+
+# used for testing sanitycheck report functionality
```

Comment:
what functionality of pandas are you using?

Reply:
I am checking that the csv file is not empty (using pandas) after the execution of the report functions. Also want to assert on the list of column headers. The easy way to do it is through pandas. Did not find a function like dataframe.empty in csv module.

Reply:
you do not need pandas for this, open it with regular csv module and look at the results, i.e. it will provide a list which would be empty of the file is empty...

Reply:
I am using pandas also to remove the NA values and then asserting on the values of the csv dataset. Let me know if it is okay to use pandas.

Reply:
this is not a good reason to add yet another dependency, let me know why cvs module is not enough.